commands/channel.go

```diff
@@ -356,18 +357,23 @@ func unarchiveChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) e
 		return errors.New("enter at least one channel")
 	}
 
+	var result error
 	channels := getChannelsFromChannelArgs(c, args)
 	for i, channel := range channels {
 		if channel == nil {
-			printer.PrintError("Unable to find channel '" + args[i] + "'")
+			partialError := fmt.Sprintf("Unable to find channel '" + args[i] + "'")
```

Comment:
`channels` and `args` are not correlated with indexes, you can't use `args[i]` to find channel names. The `args` are separated with a whitspace whereas channels are separated with a colon (`:`).